src/components/ErrorBoundary/ErrorFallback.tsx

```diff
@@ -25,11 +24,11 @@ const ErrorFallback = () => (
 
 export default ErrorFallback;
 
-const StyledHeader = styled<H1Type>(H1)`
+const StyledHeader = styled(H1)`
```

Comment:
Should these use the StyledComponent pattern the other components use?

Reply:
I added that assertion specifically where type errors were thrown. This is what we were doing previously with `H1Type` etc. so I think it's sufficient.